docs/resources/elasticsearch.md.erb

```diff
@@ -0,0 +1,123 @@
+---
+title: The Elasticsearch Resource
+---
+
+# elasticsearch
+
+Use the Elasticsearch InSpec audit resource to test the Elasticsearch configuration
+settings. The data is extracted from the live elasticsearch cluster using the curl
+command `curl 'ES_IP':'ES_PORT'/_nodes/`. The resource makes uses of FilterTable and
+its functionality can be used.
+
+## Syntax
+
+An `elasticsearch` resource is called with the elasticsearch IP and port;
+Default IP:localhost; Port:9200
+
+    describe elasticsearch(ES_IP,ES_PORT) do
+      its('property') { should eq value }
+    end
+
+where
+
+* `.where()` filter may specify a specific item and value, to which the matchers are compared
+* `.entries()` filter will return the entries that satisfy the where() condition
+* `'cluster_name'`, `'node_id'`, and `'node_name'` etc. are valid filters for this resource
+* `'property'` is a configuration setting to be tested
+* `{ should eq 'value' }` is the value that is expected
+
+## Supported Properties
+    * `'cluster_name'`, `'node_id'`, `'node_name'`,`'host'`, `'ip'`, `'version'`, `'build_hash'`
```

Comment:
Can you just make each item a single bullet item, alphabetized? Makes it easier to read through and easier for a user to find what they're looking for.